### PR TITLE
[DOCU-2778] Rework installation doc support tables and overview pages

### PR DESCRIPTION
**Pull request by lena-larionova**
### Summary
* Moved installation support tables into a support section in both Gateway and Konnect docs
* Set OpenShift to "supported"
* Retitle "Production Ready" to "Officially Supported"
* Rephrase and make notes explaining "Officially Supported" easier to notice.
* Add icons to the installation data files and generate install overview pages with tiles from them
* Create an include for install overview pages
* Add "Tech preview" badges to AWS and Azure options in Konnect

### Reason
DOCU-2778

### Testing
Gateway:
* [Supported Installation Options](https://deploy-preview-4747--kongdocs.netlify.app/gateway/latest/install-support/)
* [Installation overview](https://deploy-preview-4747--kongdocs.netlify.app/gateway/latest/install/)

Konnect:
* [Supported Installation Options](https://deploy-preview-4747--kongdocs.netlify.app/konnect/install-support/)
* [Installation overview](https://deploy-preview-4747--kongdocs.netlify.app/konnect/runtime-manager/runtime-instances/)

### CODE REVIEW

File: app/_data/tables/install_options.yml [hunk 1]
```diff
@@ -13,50 +13,60 @@ features:
         oss: true
         enterprise: true
         support: true
+        icon: /assets/images/icons/documentation/docker.png
```
Comment (mheap) [inline app/_data/tables/install_options.yml:16]: Is there an Alpine logo that we can use? This is the package, not the docker image

File: app/_data/tables/install_options.yml [hunk 2]
```diff
@@ -13,50 +13,60 @@ features:
...
 
   - name: "Docker"
     items:
       - name: "Debian (bullseye-slim)"
         oss: true
         enterprise: true
         support: true
+        url: /gateway/VERSION/install/docker/
+        icon: /assets/images/icons/documentation/docker.png
       - name: "RHEL (8-ubi)"
```
Comment (mheap) [inline app/_data/tables/install_options.yml:70]: RHEL should have a logo for Docker usage too

**Review by mheap — COMMENTED** [review]
Are we intending the index page to contain links to all installation options (supported or not)?

If so, there are some missing items. If not, how do we decide which to add a tile for?

**Comment by mheap** [comment]
> Are there other installation options you think we should link to?

RHEL and Alpine on Docker are missing based on the existing list

**Comment by lena-larionova** [comment]
> RHEL and Alpine on Docker are missing based on the existing list

Ack - I didn't realize that the "build custom images" page mentions both of those. Will add links.

File: app/_data/tables/install_options.yml (hunk 2 above)
Comment (lena-larionova) [reply app/_data/tables/install_options.yml:70]: Do you think we should do Docker icons or distro icons here? I put distro, but I could just do Docker icons for all three.

Comment (mheap) [reply app/_data/tables/install_options.yml:70]: Distro icons look good to me

**Review by mheap — APPROVED** [review]
Ship once the smoke tests are resolved (URL needs updating in the test)

**Pull request merged**